testdata/p4_16_samples_outputs/issue4883_dup_has_returned_decl2.p4-stderr

```diff
@@ -0,0 +1,6 @@
+issue4883_dup_has_returned_decl2.p4(34): [--Wwarn=uninitialized_use] warning: s.f may be uninitialized
```

Comment:
Nit: This seems like noise which can be fixed by initializing s, making it part of C2's parameters. Feel free to ignore this if required for the test.

Reply:
I do not remember if it is required for the test. @oleg-ran-amd If you think it's not required, feel free to push such changes to this branch.